src/mongo/db/repl/rs_rollback.cpp

```diff
@@ -125,7 +125,7 @@ namespace {
 
 class RSFatalException : public std::exception {
 public:
-    RSFatalException(std::string m = "replica set fatal exception") : msg(m) {}
+    RSFatalException(const std::string& m = "replica set fatal exception") : msg(m) {}
```

Comment:
Much like above, you could just leave the parameter as by-value, and then write the initailizer for msg as ```msg(std::move(m))```.